quickwit/quickwit-codegen/example/src/codegen/hello.rs

```diff
@@ -604,6 +670,18 @@ where
             })
             .map_err(|error| error.into())
     }
+    async fn check_connectivity(&mut self) -> anyhow::Result<()> {
+        if *self.num_connections_rx.borrow() == 0 {
+            anyhow::bail!("No connection to the server")
+        }
+        Ok(())
+    }
+    fn uris(&self) -> Vec<quickwit_common::uri::Uri> {
+        vec![
+            quickwit_common::uri::Uri::from_well_formed(&
+            format!("grpc://{}.service.cluster", "Hello"))
```

Comment:
Ideally, there should be as many URIs as connections in the pool. Each URI should be `grpc://<host:port>/<proto package name>.<proto service name>`.